src/sagas/index.js

```diff
@@ -0,0 +1,126 @@
+import axios from 'axios'
+import { put, takeEvery, takeLatest, all } from 'redux-saga/effects'
+
+import {
+  FETCHED_TREND,
+  FETCH_TREND,
+  ADD_FAVORITE,
+  ADDED_FAVORITE,
+  DELETE_FAVORITE,
+  DELETED_FAVORITE,
+  FETCH_DETAILS,
+  FETCHED_DETAILS,
+  SEARCH_ON_WEB,
+  SEARCHED_ON_WEB
+} from '../actions/types'
+
+const API_KEY = 'rqQsxVnE0vVW4UVFNqfjHpgjMijSundV'
+const BASE_URL = 'https://api.giphy.com'
+
+function* helloSaga() {
+  yield console.log('Hello Sagas!')
+}
+
+export function* fetchTrend() {
+  const PATH = '/v1/stickers/trending'
+  const response = yield axios.get(
```

Comment:
This `axios.get` should be called inside a `call` effect.  To avoid invoking call effect in the saga implementation. Also I would organize it a bit more and create an "api utils" that would handle this fetching.

Reply:
👍